### PR TITLE
feat: AWS Terraform provider 4.x

### PR DESCRIPTION
**Pull request by npalm**
Branch to merge PR to support Terraform Provider 4.x

See #1938

### CODE REVIEW

**Review by ScottGuymer — APPROVED** [review]

**Comment by npalm** [comment]
@ScottGuymer PR is ready to be reviewed

File: examples/default/main.tf [hunk 1]
```diff
@@ -33,11 +33,11 @@ module "runners" {
...
   # Grab zip files via lambda_download
-  # webhook_lambda_zip                = "lambdas-download/webhook.zip"
-  # runner_binaries_syncer_lambda_zip = "lambdas-download/runner-binaries-syncer.zip"
-  # runners_lambda_zip                = "lambdas-download/runners.zip"
+  webhook_lambda_zip                = "lambdas-download/webhook.zip"
+  runner_binaries_syncer_lambda_zip = "lambdas-download/runner-binaries-syncer.zip"
+  runners_lambda_zip                = "lambdas-download/runners.zip"
 
-  enable_organization_runners = true
+  enable_organization_runners = false
```
Comment (ScottGuymer) [inline examples/default/main.tf:40]: Any reason to change this in the ecample?

File: modules/runners/main.tf [hunk 2]
```diff
@@ -62,11 +62,11 @@ resource "aws_launch_template" "runner" {
       device_name = lookup(block_device_mappings.value, "device_name", "/dev/xvda")
```
Comment (ScottGuymer) [inline modules/runners/main.tf:62]: Can we remove the lookup here too?

File: examples/default/main.tf (hunk 1 above)
Comment (npalm) [reply examples/default/main.tf:40]: Yes, swapped it recently by accicdent to false. Just revert back to earlier state.

**Pull request merged**